### ISSUE TITLE
Unable to create task of type 'ShadowJar' with Gradle 2.11 rc 1

### ISSUE THREAD

**Issue by valdisrigdon** [johnrengelman/shadow#177]
Stacktrace:

```
Caused by: org.gradle.api.tasks.TaskInstantiationException: Could not create task of type 'ShadowJar'.
	at org.gradle.api.internal.project.taskfactory.TaskFactory$1.call(TaskFactory.java:123)
	at org.gradle.api.internal.project.taskfactory.TaskFactory$1.call(TaskFactory.java:118)
	at org.gradle.api.internal.AbstractTask.injectIntoNewInstance(AbstractTask.java:143)
	at org.gradle.api.internal.project.taskfactory.TaskFactory.create(TaskFactory.java:118)
	at org.gradle.api.internal.project.taskfactory.TaskFactory.createTask(TaskFactory.java:77)
	at org.gradle.api.internal.project.taskfactory.AnnotationProcessingTaskFactory.createTask(AnnotationProcessingTaskFactory.java:100)
	at org.gradle.api.internal.project.taskfactory.DependencyAutoWireTaskFactory.createTask(DependencyAutoWireTaskFactory.java:39)
	at org.gradle.api.internal.tasks.DefaultTaskContainer.create(DefaultTaskContainer.java:62)
	at org.gradle.api.internal.tasks.DefaultTaskContainer.create(DefaultTaskContainer.java:97)
	at org.gradle.api.tasks.TaskContainer$create.call(Unknown Source)
	at com.github.jengelman.gradle.plugins.shadow.ShadowJavaPlugin.configureShadowTask(ShadowJavaPlugin.groovy:40)
	at com.github.jengelman.gradle.plugins.shadow.ShadowJavaPlugin.apply(ShadowJavaPlugin.groovy:35)
	at com.github.jengelman.gradle.plugins.shadow.ShadowJavaPlugin.apply(ShadowJavaPlugin.groovy)
	at org.gradle.api.internal.plugins.ImperativeOnlyPluginApplicator.applyImperative(ImperativeOnlyPluginApplicator.java:35)
	at org.gradle.api.internal.plugins.RuleBasedPluginApplicator.applyImperative(RuleBasedPluginApplicator.java:43)
	at org.gradle.api.internal.plugins.DefaultPluginManager.doApply(DefaultPluginManager.java:137)
	... 98 more
Caused by: java.lang.NoSuchMethodError: org.gradle.api.java.archives.internal.DefaultManifest.<init>(Lorg/gradle/api/internal/file/FileResolver;)V
	at com.github.jengelman.gradle.plugins.shadow.tasks.DefaultInheritManifest.<init>(DefaultInheritManifest.groovy:15)
	at com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar.<init>(ShadowJar.java:44)
	at com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar_Decorated.<init>(Unknown Source)
	at org.gradle.api.internal.DependencyInjectingInstantiator.newInstance(DependencyInjectingInstantiator.java:48)
	at org.gradle.api.internal.ClassGeneratorBackedInstantiator.newInstance(ClassGeneratorBackedInstantiator.java:36)
	at org.gradle.api.internal.project.taskfactory.TaskFactory$1.call(TaskFactory.java:121)
```

**Comment by alkemist** [comment]
the `DefaultManifest` constructor has changed: https://github.com/gradle/gradle/blob/3d67a9b7012b28d346ea0a40c68cfe70951e37ad/subprojects/platform-jvm/src/main/java/org/gradle/api/java/archives/internal/DefaultManifest.java#L44-L44

You can get this type (`PathToFileResolver`) from the service registry just like `FileResolver`.

**Comment by mcandre** [comment]
+1, please support both Gradle v2.11 and below if it's not too much trouble

**Comment by johnrengelman** [comment]
@mcandre the latest version of Shadow (v1.2.3) supports all current released versions of Gradle from 1.12+
